whole-tale/deploy-dev
# Fixed non-ASCII characters in register_ligo.py

### Problem
LIGO Tale fails to register with a few minor formatting errors:
```
$ python ./register_ligo.py 
  File "./register_ligo.py", line 164
SyntaxError: Non-ASCII character '\xe2' in file ./register_ligo.py on line 164, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details

```

Fixes #21

### Approach
Use ASCII hyphens instead of whatever these were...

### How to Test
1. Run `python ./register_ligo.py`
    * You should see no errors
    * NOTE: No output is printed on success
